include/cantera/kinetics/InterfaceRate.h

```diff
@@ -440,7 +550,11 @@ class StickingRate : public RateType, public StickingCoverage
     double evalFromStruct(const DataType& shared_data) const {
         double out = RateType::evalRate(shared_data.logT, shared_data.recipT) *
             std::exp(std::log(10.0) * m_acov - m_ecov * shared_data.recipT + m_mcov);
-
+        if (m_chargeTransfer) {
+            // the physical interpretation of a sticking charge transfer reaction
+            // remains to be resolved.
```

Comment:
Is this worth marking with `@todo`? That way, it will turn up in searches for that (even if comments within methods don't get added to the HTML list of todo items)